toolkits/search/arcade_search/tools/google_jobs.py

```diff
@@ -0,0 +1,57 @@
+from typing import Annotated, Optional
+
+from arcade.sdk import ToolContext, tool
+from serpapi import Client as SerpClient
+
+from arcade_search.constants import DEFAULT_GOOGLE_JOBS_LANGUAGE
+from arcade_search.exceptions import LanguageNotFoundError
+from arcade_search.google_data import LANGUAGE_CODES
+
+
+@tool(requires_secrets=["SERP_API_KEY"])
+async def search_jobs(
+    context: ToolContext,
+    query: Annotated[str, "Search query"],
+    location: Annotated[
+        Optional[str],
+        "Simulates a user searching for jobs in this specific location. Defaults to None.",
+    ] = None,
+    language: Annotated[
+        str,
+        "2-letter language code to use in the Google Jobs search. "
+        f"Defaults to '{DEFAULT_GOOGLE_JOBS_LANGUAGE}'.",
+    ] = DEFAULT_GOOGLE_JOBS_LANGUAGE,
+    limit: Annotated[int, "Number of results to retrieve"] = 10,
```

Comment:
feels like this could be 25?

Reply:
you could always paginate if you're feeling up to it https://serpapi.com/google-jobs-api#api-parameters-pagination-next-page-token